src/components/voice/VoiceCall.tsx

```
@@ -58,116 +144,337 @@ function DeviceSettings({ onClose }: { onClose: () => void }) {
+                {cameraConfig[2].toFixed(1)}
+              </span>
+            </label>
+            <div className="grid grid-cols-3 gap-2">
+              <div>
+                <span className="text-[10px] text-gray-500 block text-center">
+                  左右
+                </span>
+                <input
+                  type="range"
+                  min="-2"
+                  max="2"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)